examples/networking/ip_header_monitor.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/python
+#
+# ip_header_monitor outputs every value in the ip header
+# how to use : ip_header_monitor.py <net_interface> 
```

Comment:
this is only support ipv4. You can make a note here this is ipv4 only. Later people can add ipv6 support.

what is your motivation for this example? Did you use it in your work? I try to catch the use case here.